src/lib/components/proposals/ProposalSelector.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+    import InputSelect from '$lib/components/ui/InputSelect.svelte';
+
+    // These types correspond with IC Proposal Documentation (https://wiki.internetcomputer.org/wiki/Network_Nervous_System#Proposals)
+    export type ProposalType = 'Motion' | 'AddOrRemoveNodeProvider';
+
+    export let proposalType: ProposalType | undefined;
+
+    const proposalOptions = [
+        { value: undefined, label: 'Select an option...' }, 
```

Comment:
Once the user has selected a proposal action, do you want they to be able to select empty/nothing/undefined?

if no, which I believe should be the case, undefined should only be available when the value is not yet selected.

Can you add a TODO? We can have a look afterwards no in this PR.